### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by charlescrain**
These are the architecture changes, they are breaking changes for old version. Now Instance based rather than static classes. Through the user class contracts are deployed, mirroring bloc.

### CODE REVIEW

File: BlockAppsSDK/Connection.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+using System.Threading.Tasks;
+
+namespace BlockAppsSDK
+{
+    public class Connection 
+    {
+
+        public string BlocUrl { get; private set; } = "http://localhost:8000";
+        public string StratoUrl { get; private set; } = "http://strato-dev4.blockapps.net/eth/v1.2";
+
```
Comment (liorsaar1) [inline BlockAppsSDK/Connection.cs:14]: Hard coded value.  When you have time, make that a config ?

File: BlockAppsSDKTest/Accounts/AccountTests.cs [hunk 2]
```diff
@@ -11,30 +11,60 @@ namespace BlockAppsSDKTest.Accounts
...
-            ConnectionString.BlocUrl = "http://13.93.154.77:8000";
-            //Create an Account to query
-            var allAccountAddresses = await Account.GetAccountAddresses();
-            List<Task<Account>> allAccountsTask = allAccountAddresses.Select(async addr => await Account.GetAccount(addr)).ToList();
             
-            var allAccounts = (await Task.WhenAll(allAccountsTask)).ToList();
-            Assert.IsNotNull(allAccounts);
+            var charlieAccount = await AccountManager.GetAccount("219e43441e184f16fb0386afd3aed1e780632042");
+            var accountTest = await AccountManager.GetAccount("3468276182f204ebe569fa572e8cee697f3379fd");
+            Assert.IsNotNull(charlieAccount);
```
Comment (liorsaar1) [inline BlockAppsSDKTest/Accounts/AccountTests.cs:52]: Where are these coming from ?

File: BlockAppsSDK/Connection.cs (hunk 1 above)
Comment (charlescrain) [inline BlockAppsSDK/Connection.cs:14]: That's just a default value if no string is given when creating an instance.

File: BlockAppsSDKTest/Accounts/AccountTests.cs (hunk 2 above)
Comment (charlescrain) [inline BlockAppsSDKTest/Accounts/AccountTests.cs:52]: They are addresses in http://40.118.255.235:8000/users/charlie and http://40.118.255.235:8000/users/accountTest

**Pull request closed without merging**